Algorithmia/Algorithm/AlgoOption.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AlgoOptions.swift
+//  AlgorithmiaSwiftDemo
+//
+//  Created by Erik Ilyin on 11/8/16.
+//  Copyright © 2016 algorithmia. All rights reserved.
+//
+
+import Foundation
+
+enum AlgoOption {
+    case Timeout(Int)
+    case Stdout(Bool)
+    case RawOutput
+    case VoidOutput
```

Comment:
I think we should punt `output=void` for now (i.e. just remove `VoidOutput` for now). Ultimately I think we'll end up with something along the lines of your first suggestion, but there are 4 current reasons for punting:

1. We have plans to add lots of new "async" algorithm calling features and that development may affect how we want to expose `output=void` to clients.
2. At least half of our clients don't currently support `output=void` directly, and if we need to test with it we can work around this with `username/algoname?output=void`
3. I don't think we want the name `pipeVoid`. `pipeAsync` might work, but causes a name collision for the Java client, so there is still some uncertainty that we need to address when the async API work happens.
4. The current usage of `output=void` is small enough that I'd rather solve the above questions before committing to supporting this feature in more clients.